doc/radosgw/notifications.rst

```diff
@@ -161,6 +168,7 @@ Response will have the following format:
 - User: name of the user that created the topic
 - Name: name of the topic
 - EndPoinjtAddress: the push-endpoint URL
+- if endpoint URL contain user/password information, request must be made over HTTPS. Topic get request will be rejected if not 
```

Comment:
I consider it a nit, but technically should be "contains" (if you do change this, change it in all locations, otherwise, never mind)